### PR TITLE
Update bio for benhylau

### PR DESCRIPTION
**Pull request by benhylau**

### CODE REVIEW

**Review by garrying — APPROVED** [review]

**Comment by dcwalk** [comment]
Do you mind if I propose a tiny copy edit @benhylau? I think there isn't
agreement in a couple spots

**Comment by benhylau** [comment]
@dcwalk sure can you propose as comments / gh suggests pls, thanks!

File: about.md [hunk 1]
```diff
@@ -19,10 +19,10 @@ _We are grateful to the [First Nations House](https://www.studentlife.utoronto.c
 
 Our Networks organizers have hosted civic tech events and community networks workshops, created network literacy materials, and built mesh networking software for the past two years.
 
-**Benedict Lau**{:#benhylau} is an engineer working on mobile software and mesh networks. He is a contributor and organizer at [Toronto Mesh](https://tomesh.net/), currently focused on meshing with single-board computers and building deployment tools and literacy around peer-to-peer applications.
+**Benedict Lau**{:#benhylau} is an engineer telling stories of technology practices that bring communities together. He studies distributed protocols and collective governance of digital infrastructures. When he isn't "on emails", he builds passable open source tools and facilitates activities about building peer-to-peer local networks as a way to co-imagine a future equitable web. He is a member of the Hypha Worker Co-operative and core contributor at [Toronto Mesh](https://tomesh.net).
```
Comment (dcwalk) [inline about.md:22]:
```suggestion
**Benedict Lau**{:#benhylau} is an engineer who tells stories of technology practices that bring communities together. He studies distributed protocols and collective governance of digital infrastructures. When not "on email", he builds passable open source tools and facilitates activities about peer-to-peer local networks as a way to co-imagine a future equitable web. He is a member of the Hypha Worker Co-operative and a core contributor at [Toronto Mesh](https://tomesh.net).
```

Comment (dcwalk) [reply about.md:22]: @benhylau -- trying to increase agreement / streamline parallel structure of sentences.

**Comment by dcwalk** [comment]
Hey, I cherry-picked your bio update: https://github.com/ournetworks/ournetworks.ca/pull/110/commits/3875042d3cf7d5032c9c88fdf2f19d951a0c754c

Closing this in favour of #110 as I think the timeline got a little messy!

**Pull request closed without merging**